test/CiTests.ts

```diff
@@ -116,8 +116,11 @@ describe("ci tests", function() { // tslint:disable-line only-arrow-functions
             })
 
             it("ci test: " + test, async () => {
+                console.log("Waiting for editor element...")
```

Comment:
If I use `console.log` I can see the output when debugging oni now?

I was trying to fix something this weekend and this issue arised.

Reply:
This mainly impacts the integration tests - we'll be able to see the output of the `console.log` while the test was executing.

For debugging Oni, you should be able to use `console.log` today and see it visible in the output:
![oni-screenshot-1511810392185](https://user-images.githubusercontent.com/13532591/33284740-e975de8c-d364-11e7-95de-f3422849e15b.png)

To confirm I just added a quick input binding with: 
```
Oni.input.bind("<enter>", () => console.log("hi from the console"))
```
And then pressed `<enter>` in the editor and saw the output.

If you're not seeing it, one thing to check is the settings up here:
![image](https://user-images.githubusercontent.com/13532591/33284780-0e0414da-d365-11e7-9c2f-e6042face8af.png)

Sometimes I forget to clear the 'filter' or have it set only on warn/error, in which case you wouldn't see the logs. LMK if you're still not seeing them; would make debugging very painful..

Reply:
Thanks, gonna check it!